common/djangoapps/student/management/commands/_create_users.py

```diff
@@ -21,7 +28,26 @@ def create_users(
             tos_required=False
         )
 
-        (user, _, _) = do_create_account(account_creation_form)
+        user_already_exists = False
+        try:
+            (user, _, _) = do_create_account(account_creation_form)
+        except (ValidationError, AccountValidationError) as account_creation_error:
+            # It would be convenient if we just had the AccountValidationError raised, because we include a
+            # helpful error code in there, but we also do form validation on account_creation_form and those
+            # are pretty opaque.
+            try:
+                # Check to see if there's a user with our username. If the username and email match our input,
+                # we're good, it's the same user, probably. If the email doesn't match, just to be safe we will
```

Comment:
"probably" is reassuring 😅

Reply:
You could have created a user with the same default username and email but a different password or "full name"? I'm not gonna bother testing that. At that point you've dug this hole for yourself